src/status_im/feature_flags.cljs

```diff
@@ -0,0 +1,32 @@
+(ns status-im.feature-flags
+  (:require
+    [react-native.config :as config]
+    [reagent.core :as reagent]))
+
+(defn- enabled? [v] (= "1" v))
+
+(defn check-env [k] (enabled? (config/get-config k)))
+
+(def ^:private feature-flags-config
+  (reagent/atom
+   {:wallet
```

Comment:
seperated by context mostly for the ui but this can also be one keyword.
e.g :wallet.edit-default-keypair